test/validation/api/traffic_mngr/traffic_mngr.c

```diff
@@ -317,8 +319,8 @@ static uint32_t num_ifaces;
 static odp_pool_t pools[MAX_NUM_IFACES] = {ODP_POOL_INVALID, ODP_POOL_INVALID};
 
 static odp_pktio_t pktios[MAX_NUM_IFACES];
+static bool pktio_started[MAX_NUM_IFACES];
```

Comment:
Maybe this should be odp_bool_t instead of bool, as ODP seems to avoid using the latter (but I am not sure why).